### PR TITLE
fixes #5945 - API v2 roles missing data

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/views/api/v2/roles/show.json.rabl [hunk 1]
```diff
@@ -1,7 +1,3 @@
 object @role
 
 extends "api/v2/roles/main"
-
-child :filters => :filters do
```
Comment (domcleal) [inline app/views/api/v2/roles/show.json.rabl:5]: Why remove this?

Comment (isratrade) [inline app/views/api/v2/roles/show.json.rabl:5]: @domcleal, because the child node `filters` is now in main.json.rabl and show.json.rabl extends main

Comment (domcleal) [inline app/views/api/v2/roles/show.json.rabl:5]: Oh, thanks.

Do you think that might be a bit verbose in an index view, with both filters and permissions?

**Comment by dLobatog** [comment]
@isratrade I like this, #1560 works with this too, it just misses a rebase and I think it'll be ready to merge.

**Comment by isratrade** [comment]
closing, since PR #1560 was already merged and dealt with this issue.

**Pull request closed without merging**